examples/fargate-serverless/README.md

```diff
@@ -32,6 +34,13 @@ terraform init
 terraform apply
 ```
 
+To provision this example with sample app:
+
+```sh
+terraform init
+terraform apply -var deploy_sample_app=true
```

Comment:
Do we want to guide customers to run multiple applies to avoid issues like timeouts with addons as explained in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/662 ?

Reply:
Not that I've seen so far, most of the timeout related issues are because of EC2 worker nodes (self/managed nodes) may take more time than expected while the addons trying to be created at the same time.

This is more of a standardization thinking to align with all examples, but if we prefer to keep it simple with 1 apply as long as we know there are no issues/failures, that's fine too!